spec/models/candidate_interface/degree_form_spec.rb

```diff
@@ -273,38 +249,53 @@
+      end
 
-    it 'updates the provided ApplicationForm if valid' do
-      form_data[:qualification_type] = 'Masters'
-      form_data[:subject] = 'Awoo'
-      degree.assign_attributes(form_data)
+      it 'updates the provided ApplicationForm if valid' do
+        form_data[:qualification_type] = 'Masters'
+        form_data[:subject] = 'Awoo'
+        degree.assign_attributes(form_data)
 
-      expect(degree.update(application_form)).to eq(true)
-      expect(application_form.application_qualifications.degree.first)
-        .to have_attributes(form_data)
+        expect(degree.update_base(application_form)).to eq(true)
+        expect(application_form.application_qualifications.degree.first)
+          .to have_attributes(form_data)
+      end
     end
 
-    it 'updates grade for the provided ApplicationForm if other grade is given' do
-      form_data[:grade] = 'other'
-      form_data[:other_grade] = 'Distinction'
-      degree.assign_attributes(form_data)
+    describe '#update_grade' do
+      it 'updates grade for the provided ApplicationForm if other grade is given' do
+        form_data[:grade] = 'other'
+        form_data[:other_grade] = 'Distinction'
+        degree.assign_attributes(form_data)
 
-      expect(degree.update(application_form)).to eq(true)
-      expect(application_form.application_qualifications.degree.first)
-        .to have_attributes(grade: 'Distinction')
+        expect(degree.update_grade(application_form)).to eq(true)
+        expect(application_form.application_qualifications.degree.first)
+          .to have_attributes(grade: 'Distinction')
+      end
+
+      it 'updates grade and predicted grade for the provided ApplicationForm if predicted grade is given' do
+        form_data[:grade] = 'predicted'
+        form_data[:predicted_grade] = 'First'
+        degree.assign_attributes(form_data)
+
+        expect(degree.update_grade(application_form)).to eq(true)
+        expect(application_form.application_qualifications.degree.first)
+          .to have_attributes(grade: 'First', predicted_grade: true)
+      end
     end
 
-    it 'updates grade and predicted grade for the provided ApplicationForm if predicted grade is given' do
-      form_data[:grade] = 'predicted'
-      form_data[:predicted_grade] = 'First'
-      degree.assign_attributes(form_data)
+    describe '#update_year' do
+      it 'updates year for the provided ApplicationForm if other year is given' do
+        form_data[:award_year] = '2000'
+        degree.assign_attributes(form_data)
 
-      expect(degree.update(application_form)).to eq(true)
-      expect(application_form.application_qualifications.degree.first)
-        .to have_attributes(grade: 'First', predicted_grade: true)
+        expect(degree.update_year(application_form)).to eq(true)
+        expect(application_form.application_qualifications.degree.first)
+          .to have_attributes(award_year: '2000')
+      end
     end
   end
 
```

Comment:
Missing `describe '#update_grade'` I think

Reply:
Its just above :) https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/d126832d9ff9d263020b4f7dd0d3e6aacf594008/spec/models/candidate_interface/degree_form_spec.rb#L268

Reply:
I checked twice but missed it! 👍